### DISCUSSION TITLE
Problem with installing Langflow on AWS Ubuntu instance

### DISCUSSION THREAD

**Discussion by vaderhammer** [langflow-ai/langflow#3424] [General]
Hi. I'm trying to deploy on AWS instance. I proceed acording to description and everything goes well. I'm using Ubuntu [instead of AWS linux] and maybe here is a problem. 
when i do 
--- pip install langflow, 
the instalation runs fine until it shows:
--- Building wheel for chroma-hnswlib (pyproject.toml) ... \

and it stops there. i left instance running for 12 hours and it dosconnected me, but after rebooting i receive message that langflow is not installed. I removed this chroma from reqiurements in hope that it will not install it again, but it runs again and stopped in the same place.

What do I do wrong? The instance seems to be hanged at this point, but CPU monitoring shows it's running so it's doing something
I have read this from another discussion:
https://philip.yurchuk.com/software/langflow-on-aws-ec2/
but for him Langflow actually installed, for me it's not.

**Comment by OliWongDev** [comment]
@vaderhammer I'm trying to do something similar at the moment actually and tried using EC2 + through reading the link's discussion. 

I came across this too and like he says in the link my t2.micro instance (free tier) was not a high enough capacity for it to install properly. I tested out with a larger one and it seems to get past any terminal issues; my theory (bit of newbie) is that it crashed due to not having enough power. 

Let me know how you go though, as we were trying to create a Langflow instance on AWS that allows collaboration between developers :)

**Reply by vaderhammer** [reply]
I think you are right. I checked disk usage in the instance and it seems that I had some 640MB free space. So it is in my opinion this issue. I gave up for the moment and started learning Docker. I installed Docker dektop and learn how to assemble full image locally, and then just install docker on AWS, pull image und run it using Docker container. This way I would be able to simulate running Langflow locally on Docker and then I'll try again with AWS
